src/segments/unity.go

```diff
@@ -19,6 +19,30 @@ type Unity struct {
 	CSharpVersion string
 }
 
+var csharpVersionsByUnityVersion = map[string]string{
```

Comment:
can we add this in `GetCSharpVersion()`? That way the map isn't loaded into memory if you're not using the segment.